package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
package-lock.json should not be pushed unless something is changed on package.json

Reply:
Revert and push.